kie-server-parent/kie-server-services/kie-server-services-common/pom.xml

```diff
@@ -17,6 +17,43 @@
     <java.module.name>org.kie.server.services.common</java.module.name>
   </properties>
 
+  <dependencyManagement>
+  	<dependencies>
+      <dependency>
+        <groupId>io.fabric8</groupId>
+        <artifactId>kubernetes-client-bom</artifactId>
```

Comment:
could we please create new module (kie-server-services-openshift) that will bring in all dependencies required in OpenShift and the needed code. I'd like to not embed all fabric8/kubernetes dependencies in non openshift version of kie server to avoid any potential issues with versions or libraries

for openshift image of kie server that module then will be added when image is build (and its dependencies) and for all other use case it will be completely up to developers to add it.